ilastik/applets/dataSelection/addFileButton.py

```diff
@@ -62,32 +61,19 @@ def __init__(self, parent, *, index=None, new=False):
             new (bool): Indicating if this button is used to add new lanes or files to new roles
             corresponding to an existing lane (such as prediction maps)
         """
-        super(AddFileButton, self).__init__(
-            QIcon(FILEPATH + "/../../shell/gui/icons/16x16/actions/list-add.png"),
-            "Add..." if new == False else "Add New...",
-            parent,
-        )
+        super(AddFileButton, self).__init__(QIcon(str(_ICON_PATH)), "Add New..." if new else "Add...", parent)
 
-        self._index = index
+        self.index = index
         # drop down menu for different add options
         menu = QMenu(parent=self)
-        menu.addAction("Add separate Image(s)...").triggered.connect(self.addFilesRequested.emit)
-        menu.addAction("Add a single 3D/4D Volume from Sequence...").triggered.connect(self.addStackRequested.emit)
+        menu.addAction("Add separate Image(s)...").triggered.connect(self.addFilesRequested)
+        menu.addAction("Add multiple Image(s)...").triggered.connect(self.addFromPatternsRequested)
```

Comment:
A more descriptive name would be cool. The button above already enables adding of multiple images...

Reply:
What name do you suggest?